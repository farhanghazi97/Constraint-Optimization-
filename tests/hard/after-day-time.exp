m1:mon 1pm
m2:tue 1pm
m3:wed 1pm
m4:thu 1pm
cost:0
