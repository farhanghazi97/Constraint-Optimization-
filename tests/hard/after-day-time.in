meeting, m1
meeting, m2
meeting, m3
meeting, m4
domain, m1, after mon 12pm, hard
domain, m2, after tue 12pm, hard
domain, m3, after wed 12pm, hard
domain, m4, after thu 12pm, hard
