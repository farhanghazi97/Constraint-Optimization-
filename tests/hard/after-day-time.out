m1:thu 4pm
m2:thu 4pm
m3:thu 4pm
m4:fri 4pm
cost:0
